homeassistant/components/version/__init__.py

```diff
@@ -18,16 +20,32 @@
 )
 from .coordinator import VersionDataUpdateCoordinator
 
+_LOGGER = logging.getLogger(__name__)
+
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up the version integration from a config entry."""
+
+    board = entry.data[CONF_BOARD]
+
+    # Operating system changed the name from "intel-nuc" to "generic-x86-64"
+    if board == "Intel NUC":
+        entry.data[CONF_BOARD] = "Generic x86-64"
+        hass.config_entries.async_update_entry(entry, data=entry.data)
+
```

Comment:
I do not agree with this change.
While it replaces it, we can not make the assumption that the user set up this to track their own system.
The update entities provided by the hassio integration are far superior for that purpose.

Reply:
Even its setup to track someone elses system, its Generic x86-64 what they be using now :man_shrugging: 

We can also drop it, works for me.